client/me/purchases/payment-methods/components/payment-method-edit-form-fields.tsx

```diff
@@ -0,0 +1,93 @@
+import { getCountryPostalCodeSupport } from '@automattic/wpcom-checkout';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+import CountrySelectMenu from 'calypso/my-sites/checkout/composite-checkout/components/country-select-menu';
+import FormFieldAnnotation from 'calypso/my-sites/checkout/composite-checkout/components/form-field-annotation';
+import {
+	LeftColumn,
+	RightColumn,
+} from 'calypso/my-sites/checkout/composite-checkout/components/ie-fallback';
+import useCountryList from 'calypso/my-sites/checkout/composite-checkout/hooks/use-country-list';
+
+const GridRow = styled.div`
+	display: -ms-grid;
+	display: grid;
+	width: 100%;
+	-ms-grid-columns: 48% 4% 48%;
+	grid-template-columns: 48% 48%;
+	grid-column-gap: 4%;
+	justify-items: stretch;
+`;
+
+const FieldRow = styled( GridRow )`
+	margin-top: 16px;
+
+	:first-of-type {
+		margin-top: 0;
+	}
+`;
+
+const RenderEditFormFields = ( {
+	onChangePostalCode,
+	onChangeCountryCode,
+	postalCodeValue,
+	countryCodeValue,
+}: {
+	value?: string;
+	onChangePostalCode: unknown;
+	onChangeCountryCode: unknown;
+	postalCodeValue?: string;
+	countryCodeValue?: string;
+} ): JSX.Element => {
+	const countriesList = useCountryList( [] );
+	const translate = useTranslate();
+
+	const arePostalCodesSupported = getCountryPostalCodeSupport(
+		countriesList,
+		countryCodeValue ?? ''
+	);
+
+	if ( ! postalCodeValue ) {
+		postalCodeValue = '';
+	}
+
+	return (
+		<>
+			<FieldRow>
+				<LeftColumn>
+					<CountrySelectMenu
+						translate={ translate }
+						onChange={ onChangeCountryCode }
+						isError={ '' }
```

Comment:
Minor:

```suggestion
						isError={ false }
```

(Really, this should be optional but `CountrySelectMenu` is JS and since there are no default values, it looks like all its args are required. Not something that needs to be dealt with here.)

Reply:
Fixed.